src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order.yml

```diff
@@ -78,7 +78,7 @@ winzou_state_machine:
                     on: ["cancel"]
                     do: ["@sm.callback.cascade_transition", "apply"]
                     args: ["object", "event", "'cancel'", "'sylius_order_shipping'"]
-                sylis_cancel_order:
+                sylius_cancel_order:
```

Comment:
While it's great not to have any typo in callback name, I'd hold it until we figure out what should we do in those cases (there's also #9661).

This fix would break external applications in which this callback was previously customised or disabled. It's a rare edge case, but a possible one.

Reply:
Potential BC indeed. Isn't there an option to have them both in there and deprecate the 'wrong' named one (e.g. with a deprecation service call added to it via a listener or whatever)? Then remove them in an upcoming version.